File: cpp_solutions/TwoSum.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& numbers, int desireResult){
        for(int i=0; i<numbers.size(); i++){
            for(int j=i+1; j<numbers.size(); j++){
                if (numbers[i]+numbers[j]== desireResult){
                    return {i,j};
                }
            }
        }
        return {};
    }
};

File: python_solutions/TwoSum.py
class Solution:
    def twoSum(self, numbers: List[int], desireResult: int) -> List[int]:
        ListOfNums = {}
        for i, n in enumerate(numbers):
            complement = desireResult - n
            if complement in ListOfNums:
                return [ListOfNums[complement], i]
            ListOfNums[n] = i
        return []